aws_lambda_powertools/utilities/parameters/appconfig.py

```diff
@@ -104,7 +104,7 @@ def _get(self, name: str, **sdk_options) -> str:
         sdk_options["ClientId"] = CLIENT_ID
 
         response = self.client.get_configuration(**sdk_options)
-        return response["Content"].read()  # read() of botocore.response.StreamingBody
+        return response["Content"].read().decode("utf-8")  # read() of botocore.response.StreamingBody
```

Comment:
oh wait, I missed this part as I just noticed the PR was showing outdated. Doesn't this break what consumers are already expecting despite the incorrect type?

Our tests were expecting bytes before.

Reply:
yes, you are correct.